pkg/initializer/initializer.go

```diff
@@ -90,14 +88,21 @@ func (e *EtcdInitializer) Initialize(mode validator.Mode, failBelowRevision int6
 	metrics.ValidationDurationSeconds.With(prometheus.Labels{metrics.LabelSucceeded: metrics.ValueSucceededTrue}).Observe(time.Since(start).Seconds())
 
 	if dataDirStatus != validator.DataDirectoryValid {
-		start := time.Now()
-		restored, err := e.restoreCorruptData()
-		if err != nil {
-			metrics.RestorationDurationSeconds.With(prometheus.Labels{metrics.LabelSucceeded: metrics.ValueSucceededFalse}).Observe(time.Since(start).Seconds())
-			return fmt.Errorf("error while restoring corrupt data: %v", err)
-		}
-		if restored {
-			metrics.RestorationDurationSeconds.With(prometheus.Labels{metrics.LabelSucceeded: metrics.ValueSucceededTrue}).Observe(time.Since(start).Seconds())
+		if dataDirStatus == validator.DataDirStatusInvalidInMultiNode || (e.Validator.OriginalClusterSize > 1 && dataDirStatus == validator.DataDirectoryCorrupt) {
+			e.restoreInMultiNode()
+		} else if e.Validator.OriginalClusterSize > 1 && isPresent {
+			e.restoreInMultiNode()
```

Comment:
there are no grafana is available for Multi-node, so spending time on this thing ... doesn't make sense to me.

Reply:
I think you can add it already in this PR, unless you can create a separate issue for tracking the addition of new prometheus metrics for multi-node etcd (so that we don't miss out such minor cases)

Reply:
Isn't we already have an issue open which will take care of all the metrices required for multi-node: https://github.com/gardener/etcd-druid/issues/358

Reply:
Ok. Can we start populating that issue? Like a checklist of metrics to be added. That will ensure that we don't miss this particular one, or many others that should be part of multi-node etcd PRs.

Reply:
make sense. done https://github.com/gardener/etcd-druid/issues/358#issuecomment-1198917054